fastq/pairedEnd.go

```diff
@@ -46,15 +46,15 @@ func PairEndToChan(readOne string, readTwo string, output chan<- *PairedEnd) {
 	close(output)
 }
 
-func ReadPairBigToChan(readOne string, readTwo string, answer chan<- *PairedEndBig) {
-	var fq *PairedEndBig
+func ReadPairBigToChan(readOne string, readTwo string, answer chan<- PairedEndBig) {
```

Comment:
I think we are both making changes here.  I also needed to change this to get some testing in SimpleGraph to run.  I initially made the same change you did, but after a quick back-and-forth with Eric I added in some code he had, which gave him better performance.  I think we just want to watch this function as we both merge in our changes.